docs/source/index.rst

```diff
@@ -1,53 +1,56 @@
 Welcome to sparkly's documentation!
 ===================================
 
-Sparkly is a lib which makes usage of pyspark more convenient and consistent.
+Sparkly is a lib that makes usage of pyspark more convenient and consistent.
```

Comment:
[optional] lib -> library or python package, I know this line was introduced by me, but now it looks bad to me.